<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">  background-color: #4070f4;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">border-radius:6px;
background-color:white;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QFrame" name="frame_7">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="titulo">
           <property name="styleSheet">
            <string notr="true">font: 11pt &quot;Bahnschrift Light SemiCondensed&quot;;
margin: 5px 0 0 0;
</string>
           </property>
           <property name="text">
            <string>Asignar Internet</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">
font: 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
</string>
           </property>
           <property name="text">
            <string>Ingrese la dirección IP de la computadora que desea asignarle Internet</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>131</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_ip">
              <property name="styleSheet">
               <string notr="true">QLineEdit{
font: 25 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
outline:none;
border:1px solid #aaa;
padding:5px;
border-radius: 5px;
color: #333;
}

QLineEdit:focus{
border: 1px solid #007acc;

}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="placeholderText">
               <string>Ingrese la dirección IP</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>131</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_6">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Hora Inicio de Navegación</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="date_hora_inicio">
              <property name="styleSheet">
               <string notr="true">
font: 25 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
outline:none;
border:1px solid #aaa;
padding:5px;
border-radius: 5px;
color: #333;
margin:5px 0 0 0;

</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="styleSheet">
            <string notr="true">margin: 0 10px 0 0;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>MAC</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_mac">
              <property name="styleSheet">
               <string notr="true">QLineEdit{
font: 25 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
outline:none;
border:1px solid #aaa;
padding:5px;
border-radius: 5px;
color: #333;
margin:5px 20px 0 20px;
}

QLineEdit:focus{
border: 1px solid #007acc;

}
</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="placeholderText">
               <string>Ingrese la MAC</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Hora Final de Navegacion</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="date_hora_final">
              <property name="styleSheet">
               <string notr="true">
font: 25 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
outline:none;
border:1px solid #aaa;
padding:5px;
border-radius: 5px;
color: #333;
margin:5px 0 0 0;

</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_8">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="respuesta">
           <property name="styleSheet">
            <string notr="true">
font: 9pt &quot;Bahnschrift Light SemiCondensed&quot;;
</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_asignar">
           <property name="styleSheet">
            <string notr="true">QPushButton{ 
  width: 100%;
  padding: 10px;
  border-radius: 6px;
  line-height: 18px;
  background-color: #4070f4;
  color:white;
font: 10pt &quot;Bahnschrift Light SemiCondensed&quot;;
margin:0 150px 10px 150px;}

QPushButton:hover{
  background-color: darkblue;}</string>
           </property>
           <property name="text">
            <string>Asignar Internet</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
